server/build.gradle

```diff
@@ -36,6 +38,11 @@ application {
     startScripts.applicationName = 'scalardb-server'
 }
 
+docker {
+    name "ghcr.io/scalar-labs/scalardb-server:$version"
+    files tasks.distTar.outputs, 'conf/database.properties.tmpl', 'conf/log4j.properties'
```

Comment:
The template file `conf/database.properties.tmpl` seems to have been removed.

Reply:
Good catch! Thank you!